docs/config.md

```diff
@@ -0,0 +1,182 @@
+# The go-ipfs config file
+The go-ipfs config file is a json document. It is read once at node instantiation,
```

Comment:
@Kubuxu: "No."

If not, why not? 

> "Because the init is init of repo, and node instantiation is when we perform any command without the daemon, or when we start the daemon. 

Can we include this, somewhere?